src/graphql/language/block_string.py

```diff
@@ -38,7 +38,7 @@ def dedent_block_string_value(raw_string: str) -> str:
 
 def is_blank(s: str) -> bool:
     """Check whether string contains only space or tab characters."""
-    return all(c == " " or c == "\t" for c in s)
+    return all(c in [" ", "\t"] for c in s)
```

Comment:
It's disputable what would be best for readability and performance here. Note that there are at least 5 alternatives that look good to me:
- `c in " \t"`
- `c == " " or c == "\t"`
- `c in [' ', '\t']`
- `c in (' ', '\t')`
- `c in {' ', '\t'}`

The first solution only works if you are sure that `c` is a single character. Note that creating a list, tuple or set has a certain cost.

Out of curiosity, I measured with `timeit` and found that in fact `set` is the fastest solution, even if you only have 2 options. I would have guessed that the effect would be only noticeable with more options. Note that you want to optimize both cases, that the condition is true and that it is false. With the set, you have always a constant time. The other solutions are slower when the condition is false, and become worse the more options you have.

Reply:
If we make that change, we should look if there are similar comparisons and change all of them.

Reply:
For the records, I found that using a regex was even faster than all other solutions, and changed it in f917259a5ddc0b8a791a398cf2a459ec532f0736